shared-helpers/src/locales/es.json

```diff
@@ -701,6 +702,7 @@
   "t.delete": "Borrar",
   "t.deposit": "Depósito",
   "t.description": "Ingrese descripción",
+  "t.download": "Download",
```

Comment:
Sorry, I can't seem to find the download translations and want to make sure this is used in Spanish too!